### DISCUSSION TITLE
Dynamic primary color

### DISCUSSION THREAD

**Discussion by jeffreyyoung** [nativewind/nativewind#471] [Q&A] [answered]
My app has a primary color (normally blue). I'd like to give users the ability to set the primary color. In normal tailwind I'd accomplish this by 
 mapping "bg-primary-500" to a css variable that is updated at runtime.  What would be the best way to accomplish this with native wind?

**Comment by DarkcoderSe** [comment]
have you find any solution?

**Comment by MPJHorner** [comment]
Always interested in this, ultimately it seems this package is creating the JSON properties at build time and it isn't possible (at the moment) to embed a function which is actually what is needed here.

(the above assumes you are trying to do this for React Native on Mobile, not web)

Will keep digging too

**Comment by MPJHorner** [comment, marked as answer]
I've saw this on v4 - https://www.nativewind.dev/v4/guides/themes

Not sure if it will be supported on mobile.

**Reply by marklawlor** [reply]
It works on mobile

**Comment by marklawlor** [comment]
The use of CSS variables as colours is one of the main features of V4. See pinned issue for more information.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
